Tasks/Common/artifacts-common/stringUtils.ts

```diff
@@ -0,0 +1,21 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export function getProjectAndFeedIdFromInputParam(inputParam: string): any {
```

Comment:
Where is this used?

Reply:
not used currently, but kept it anyway when copying from packaging-common.

Reply:
I'd remove it, personally. It doesn't do much, and IMO it's more clear when inputs are read from within the task itself when reasonable to do so.